javascript.ru/2.12.4/2.12.4.js

```diff
@@ -0,0 +1,22 @@
+﻿var user = prompt('Who are you?');
+
+if (user == 'Admin') {
+	var password = prompt('Enter password');
+	if (password == 'Loki') {
+		alert('Wellcome!');
+	}	
+	else if (password == 'Cancel') {
+		alert('Canceled');
+	}
+	else {
+		alert('Not correct password');
+	}
+}
+ 
```

Comment:
Remove this empty line. It is better not to insert line break between if and else.

Reply:
Done a9e11ec.